### DISCUSSION TITLE
How do you enable cors?

### DISCUSSION THREAD

**Discussion by nicholasq** [nitrojs/nitro#821] [Q&A]
How do we enable cors? It looks like you can enable cors with a plugin in h3 but I'm not sure how I can tell nitro to use the plugin.

**Comment by oumarbarry** [comment]
@pi0, it would be nice to have some clarity on this.

**Comment by RyanHo97** [comment]
I also want to start CORS. I have tried a series of methods, including this example，but it didn't work:

![image](https://github.com/unjs/nitro/assets/49436413/512a27aa-717b-4434-8cab-0987ddcf125f)

**Reply by RyanHo97** [reply]
I tried the method on the document at the same time but still couldn't solve it：
https://nitro.unjs.io/config
![image](https://github.com/unjs/nitro/assets/49436413/7edd515a-61a3-41af-9e85-94859825a4cf)
